src/styles/search.css

```diff
@@ -0,0 +1,35 @@
+.search-input {
+  width: 30%;
+  height: 30px;
+  font-size: 20px;
+  border: 1px solid black;
+  float: left;
+  border-top-left-radius: 10px;
+  border-bottom-left-radius: 10px;
+  font-family: "Roboto Mono", monospace;
+}
+.search-form {
+  display: flex;
+  flex-direction: row;
```

Comment:
`flex-direction` is set to `row` implicitly, you dont need to declare it like this unless you are overwriting a previous definition. Can delete this no problem